### ISSUE TITLE
Re-Evaluate Higher Explainability Models

### ISSUE THREAD

**Issue by sam-mata** [sam-mata/antarctic-ai#74]
### Details
Several models initially excluded should now be re-evaluated due to the higher degree of explainability and interpretability inherent in their architectures, compared to currently higher performing alternatives.

### Models
- [ ] LinearRegression
- [ ] Ridge
- [ ] Lasso
- [ ] ElasticNet
- [ ] TheilSenRegressor
- [ ] HuberRegressor
- [ ] DecisionTreeRegressor
- [ ] KNeighborsRegressor
- [ ] PLSRegression
- [ ] LinearSVR
- [ ] SVR
- [ ] NuSVR

**Comment by sam-mata** [comment]
| Model | Ice Thickness R2 | Ice Velocity R2 | Ice Mask R2 |
|-------|-------------------|------------------|-------------|
| Linear Regression | 0.880783 | 0.028451 | 0.814034 |
| Ridge | 0.880783 | 0.028451 | 0.814034 |
| Lasso | -0.000034 | -0.000190 | 0.165243 |
| ElasticNet | 0.172615 | -0.000190 | 0.511552 |
| Lars | 0.880783 | 0.027709 | 0.801598 |
| LassoLars | -0.000034 | -0.000190 | 0.165240 |
| OrthogonalMatchingPursuit | 0.822749 | -0.000570 | 0.404192 |
| BayesianRidge | 0.880783 | 0.028455 | 0.814034 |
| ARDRegression | 0.871622 | -0.000190 | 0.811955 |
| SGDRegressor | -3.024589e+30 | -9.341201e+32 | -1.096313e+30 |
| PassiveAggressiveRegressor | -0.016971 | -57.501461 | 0.293664 |
| HuberRegressor | 0.868030 | -0.021068 | 0.776648 |
| RANSACRegressor | -0.446284 | -0.023980 | -0.661869 |
| TheilSenRegressor | 0.859414 | -0.015187 | 0.779686 |
| LinearSVR | 0.661750 | -0.005251 | 0.586495 |
| KNeighborsRegressor | 0.833221 | -0.009148 | 0.912587 |
| DecisionTreeRegressor | 0.970125 | -0.627897 | 0.960005 |
| RandomForestRegressor | 0.988045 | 0.384788 | 0.979989 |
| ExtraTreesRegressor | 0.987739 | 0.485918 | 0.982640 |
| GradientBoostingRegressor | 0.784374 | 0.105600 | 0.801342 |
| AdaBoostRegressor | 0.895826 | 0.029123 | 0.922690 |
| MLPRegressor | -1.075557 | -1310.085758 | -0.184319 |
| PLSRegression | 0.789626 | 0.023283 | 0.781333 |
| XGBoost | 0.815272 | -0.042789 | 0.820753 |
| LightGBM | 0.812638 | 0.268137 | 0.822156 |
